draft-ietf-quic-http.md

```diff
@@ -1824,7 +1815,9 @@ CONTINUATION (0x9):
 
 Frame types defined by extensions to HTTP/2 need to be separately registered for
 HTTP/3 if still applicable.  The IDs of frames defined in {{!RFC7540}} have been
-reserved for simplicity.  See {{iana-frames}}.
+reserved for simplicity.  Note that the frame type space in HTTP/3 is
+substantially larger (62 bits versus 8 bits), so many HTTP/3 frame types have no
+equivalent HTTP/2 code point.   See {{iana-frames}}.
```

Comment:
Number agreement: `s/point/points/`

Reply:
I'm inclined to think that either is correct, since there can be many types (each of which has no equivalent code point) or a range of types (which has no corresponding range of code points).  However, I don't care that much.  :-)